docs/building-apps/network-features/vetkeys/introduction.mdx

```diff
@@ -10,89 +10,42 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+- **Encrypted**: Following standard practices in [identity-based encryption (IBE) schemes](/docs/references/vetkeys-overview), the derived vetKeys are securely transported to users in encrypted form. This ensures that neither individual nodes nor the network ever have access to a user’s private vetKeys.
 
-## vetKeys use cases and applications
-
-The availability of vetKeys allows for a series of applications, including but not limited to those covered in the following sections.
-
-### [Distributed key management service (DKMS)](/docs/building-apps/network-features/vetkeys/dkms/introduction)
-
-`DKMS` is a higher-level library built on top of `vetKeys`, designed to offer simple, developer-friendly APIs especially for those without cryptographic expertise. It allows users to securely generate, manage, and share cryptographic keys using `vetKeys` and ICP. These keys can then be used for encryption, decryption, or data signing.
-
-DKMS has two primary components:
-- **[`KeyManager`](dkms/key-manager.mdx)**: Access control and vetKey derivation.
-- **[`EncryptedMaps`](dkms/encrypted-maps.mdx)**: Built on top of the `KeyManager` for efficient data encryption and storage in a key-value store.
-
-One of the primary use cases of `DKMS` is to enable a blockchain to securely encrypt and host data at scale to support millions of users and billions of secrets. Because BLS signatures are unique, they can be directly used as symmetric keys. For example:
-
-- **Secure file storage**: In a secure file storage dapp, users can encrypt their files using `DKMS` before uploading them. The dapp ensures that only authorized users can decrypt the files. While blockchain nodes assist in key recovery, they never see the private keys or decrypted file contents.
-
-- **Secure messaging**: A secure messaging dapp can encrypt conversations using the identities of the two participants. The dapp manages access so only those users can retrieve the decryption keys.
-
-- **Decentralized social networking (SocialFi)**: In a decentralized social network, posts can be encrypted with a key tied to a unique post identifier. The SocialFi dapp ensures that only the author and selected recipients have access to the corresponding decryption key.
-
-### [Blockchain-issued BLS signatures](/docs/building-apps/network-features/vetkeys/advanced/bls-signatures)
+- **Threshold**: As vetKeys are computed in a distributed manner across ICP nodes, there is no central authority generating vetKeys for users.
 
-vetKeys can be used as signatures issued by the blockchain. This is especially useful for blockchains that don't have a built-in certification feature enabling dapps to sign statements.
+- **Keys**: vetKeys are [BLS signatures](https://learn.internetcomputer.org/hc/en-us/articles/34209540682644-Subnet-Keys-and-Subnet-Signatures), the native signature scheme on ICP, and they are unique, making them well-suited for use as cryptographic decryption keys.
```

Comment:
I would make it more general, on the line of what I suggested below. An alternative would be to have a sentence before the bullet points, saying something like "vet in VetKeys stands for:", and remove this bullet point. 

```suggestion
- **Keys**: key material that can be used to further derive strong cryptographic keys, such as encryption and signature keys.
```